hr_course/data/mail_template.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="mail_template_validity_reminder_en" model="mail.template">
+            <field name="name">Course Validity Notification</field>
+            <field name="email_from">${object.email_from}</field>
+            <field name="email_to">${object.email_to}</field>
+            <field name="subject">End of validity for a course</field>
+            <field name="model_id" ref="model_hr_course" />
+            <field name="body_html" type="html">
+                <div>
+                    <p>Hi,</p>
+                    <p
+                    >The validity time of the course ${object.name} will end up soon. You maybe need to plan a new one. </p>
+                    <p
+                    >The concerned employees are ${"\n".join([n.name for n in object.course_attendee_ids.employee_id])}</p>
```

Comment:
note for reviewer, seems that employees names aren't properly mapped here and instead we get course name:
![image](https://user-images.githubusercontent.com/59824990/141995155-4e57b907-007a-478d-bac0-47fc013e6c6d.png)
![image](https://user-images.githubusercontent.com/59824990/141995177-3316e6cd-2b5e-4d35-81a6-8e5dd23eae0e.png)